### PR TITLE
PSR-310 New error message inserted

### PR DESCRIPTION
**Pull request by AlessandroCinque**
https://jira.tools.tax.service.gov.uk/browse/PSR-310 New message inserted

### CODE REVIEW

**Comment by AlessandroCinque** [comment]
tests running fine 
image

**Review by BabuRupoll7623 — DISMISSED** [review]

File: app/forms/YesNoPageFormProvider.scala [hunk 1]
```diff
@@ -25,7 +25,7 @@ import javax.inject.Inject
 
 class YesNoPageFormProvider @Inject()() {
 
-  protected[forms] val textAreaRegex = """^[a-zA-Z0-9\-'" \t\r\n,.@/]+$"""
+  protected[forms] val textAreaRegex = """^[a-zA-Z0-9\-'" \t\r\n,./]+$"""
```
Comment (narendrapaduchuri) [inline app/forms/YesNoPageFormProvider.scala:28]: Hi, it seems need '@' sign in the regex, I am not sure why this has been removed. In the updated error message also design team expecting the 'atʼ signs (@)'

**Review by cyeniceri — APPROVED** [review]

**Review by Vamshi-Muniganti — APPROVED** [review]
LGTM

File: app/forms/YesNoPageFormProvider.scala (hunk 1 above)
Comment (Vamshi-Muniganti) [reply app/forms/YesNoPageFormProvider.scala]: @ symbol should be accepted , it should be there as part of regex

**Review by BabuRupoll7623 — APPROVED** [review]

**Pull request closed without merging**